classes/file_search.php

```diff
@@ -151,17 +151,20 @@ public static function get_criteria(files $record): object {
      *
      * @param \tool_advancedreplace\files $record
      * @param string $output path
-     * @param int $limitfrom limit for sql
-     * @param int $limitnum limit for sql
+     * @param int $startid minimum id for sql
+     * @param int $endid maximum id for sql
+     * @param bool $finalshard True if this is the ladt shard
      * @return void
      */
-    public static function files(files $record, string $output = '', int $limitfrom = 0, int $limitnum = 0) {
+    public static function files(files $record, string $output = '',
+    int $startid = 0, int $endid = 0, bool $finalshard = false) {
```

Comment:
indenting

Reply:
:(  This is how phpcbf made it.  I will fix.

Reply:
I think this still needs further indents based on https://moodledev.io/general/development/policies/codingstyle#wrapping-of-the-function-signatures